git-infer

```diff
@@ -1,11 +1,17 @@
 #!/usr/bin/env bash
 
 COMMAND="git diff --staged --name-status"
+TYPE=0
  
-while getopts ":a" opt; do
+while getopts ":an" opt; do
```

Comment:
How do you feel about `-a` for adding the files (as it is now) and `-A` for adding **a**ll files?